helm-deploy-test/tasks/cf-deploy.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+DIR_PATH=$(pwd)
+
+#export kube-host details from pool
+set -a; source pool.k8s-hosts/metadata; set +a
+
+#kube-ready-state-check script
+curl -O https://raw.githubusercontent.com/SUSE/scf/develop/bin/dev/kube-ready-state-check.sh
+
+#check kube host readiness to deploy CF
+ssh-keygen -N "" -f /root/.ssh/id_rsa
+sshpass -e ssh-copy-id -o StrictHostKeyChecking=no ${K8S_USER}@${K8S_HOST_IP}
+ssh -o StrictHostKeyChecking=no ${K8S_USER}@${K8S_HOST_IP} 'bash -s' \
+    < kube-ready-state-check.sh
```

Comment:
No need for the key, you just need one line anyway:
```bash
sshpass -e ssh -o StrictHostKeyChecking=no "${K8S_USER}@${K8S_HOST_IP}" -- \
    bash -s < kube-ready-state-check.sh
```
If there's a reason I'm missing, please add a comment.